#! /bin/sh -e
## 14_fix_kmenu.dpatch by Martin Loschwitz <madkiss@madkiss.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix kmenu displaying

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0

--- a/src/widgets/qpopupmenu.cpp	29 Apr 2004 22:31:28 -0000	1.60
+++ a/src/widgets/qpopupmenu.cpp	30 Apr 2004 01:11:59 -0000
@@ -2531,7 +2531,7 @@ QSize QPopupMenu::sizeHint() const
 
     QPopupMenu* that = (QPopupMenu*) this;
     //We do not need a resize here, just the sizeHint..
-    return that->updateSize(FALSE, FALSE).expandedTo( QApplication::globalStrut() );
+    return that->updateSize(FALSE).expandedTo( QApplication::globalStrut() );
 }
